app/src/main/java/com/khmelenko/lab/travisclient/mvp/MvpActivity.java

```diff
@@ -18,6 +19,10 @@
  */
 public abstract class MvpActivity<T extends MvpPresenter> extends AppCompatActivity {
 
+    static {
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
+    }
```

Comment:
What is the purpose of having static constructor here? I would avoid having such thing in code.